django/core/cache/backends/memcached.py

```diff
@@ -194,6 +194,56 @@ def __init__(self, server, params):
     def _cache(self):
         return self._lib.Client(self._servers, **self._options)
 
+    def add(self, key, value, timeout=DEFAULT_TIMEOUT, version=None):
+        try:
+            return super().add(key, value, timeout=timeout, version=version)
+        except (self._lib.ConnectionError, self._lib.ServerDown, self._lib.Error) as e:
+            warnings.warn(str(e), RuntimeWarning)
+            return value
+
+    def get(self, key, default=None, version=None):
```

Comment:
`get_many` and `clear` aren't subclassed here, should they be?

Also, I was wondering if it might be easier to keep in sync in the future, if the order of methods matched those from `BaseMemcachedCache`?